types/ext/language-descriptors.d.ts

```diff
@@ -113,6 +113,7 @@ type AllTextProcessors = {
         pre: CapitalizationPreprocessors & {
             removeAlphabeticDiacritics: TextProcessor<boolean>;
         };
+    lo: Record<string, never>;
```

Comment:
oops, seems this got nested into the `la` object

Reply:
Yes, so sorry.